### DISCUSSION TITLE
Not focused in last opened tab

### DISCUSSION THREAD

**Discussion by Stas1981ru** [qutebrowser/qutebrowser#5688] [Q&A] [answered]
When I click in "adverticement" new tab with adverticement site opened. It is ok.
But I still focused at a parent tab.

I think the last opened tab must be focused. How I can do it ?
---
config.val.tabs.mode_on_change is "normal"

**Comment by The-Compiler** [comment, marked as answer]
See #3231 - you can set `tabs.background` to true` - confusingly, this will open such links in a foreground tab, but open other followed links in the background.
